### PR TITLE
Completed the linguistic review.

### PR DESCRIPTION
**Pull request by RukshiW**
## Documentation

Reviewed the documentation in the following Ballerina Lang repositories:
- grpc-bidirectional-streaming
- grpc-client-streaming
- grpc-secured-unary
- grpc-server-streaming
- grpc-unary-blocking
- grpc-unary-non-blocking
- header-based-routing
- hello-world-parallel
- hello-world-service
- hello-world

The following two files were reverted due to merge conflicts. They will be re-committed.
- header-based-routing.bal
- hello-world-servive.bal

### CODE REVIEW

File: docs/ballerina-by-example/examples/grpc-bidirectional-streaming/grpc-bidirectional-streaming.description [hunk 1]
```diff
@@ -1,2 +1,2 @@
-// The gRPC Server Connector can be used to expose service gRPC service over http2.
-// This sample is to demonstrate gRPC client streaming service interacting with gRPC non-blocking client.
+// The gRPC Server Connector exposes the service named 'gRPC service' over http2.
```
Comment (uthaiyashankar) [inline docs/ballerina-by-example/examples/grpc-bidirectional-streaming/grpc-bidirectional-streaming.description]: gRPC service (a type of service).. not service named 'gRPC service'

File: docs/ballerina-by-example/examples/grpc-bidirectional-streaming/client/chat.sample.client.bal [hunk 2]
```diff
@@ -13,7 +13,7 @@ function main (string[] args) {
     };
 
     endpoint grpc:Client ep;
-    // Executing unary non-blocking call registering server message listener.
+    // The executing unary non-blocking call registering server message listener.
```
Comment (uthaiyashankar) [inline docs/ballerina-by-example/examples/grpc-bidirectional-streaming/client/chat.sample.client.bal]: Can you double check this please? sounds incomplete, may be I am mistaken.

File: docs/ballerina-by-example/examples/grpc-secured-unary/grpc-secured-unary.description [hunk 3]
```diff
@@ -1,2 +1,2 @@
-// The gRPC Server Connector can be used to expose service gRPC service over http2.
-// This sample is to demonstrate gRPC unary service secured interacting with gRPC blocking client.
+// The gRPC Server Connector exposes the 'gRPC service' service over http2.
```
Comment (uthaiyashankar) [inline docs/ballerina-by-example/examples/grpc-secured-unary/grpc-secured-unary.description]: can we remove "service" (double)..

File: docs/ballerina-by-example/examples/grpc-client-streaming/client/helloWorld.sample.client.bal [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-    // Client endpoint configuration
+    // The client endpoint configuration.
     endpoint HelloWorldClient helloWorldEp {
         host:"localhost",
         port:9090
     };
 
     endpoint grpc:Client ep;
-    // Executing unary non-blocking call registering server message listener.
+    // The executing unary non-blocking call registering server message listener:
```
Comment (afkham) [inline docs/ballerina-by-example/examples/grpc-client-streaming/client/helloWorld.sample.client.bal]: This doesn't sound correct, technically.

File: docs/ballerina-by-example/examples/grpc-client-streaming/client/helloWorld.sample.client.bal [hunk 5]
```diff
@@ -35,28 +35,28 @@ function main (string[] args) {
     }
     _ = ep -> complete();
 
-    //to hold the programme
+    //This holds the programme.
```
Comment (afkham) [inline docs/ballerina-by-example/examples/grpc-client-streaming/client/helloWorld.sample.client.bal]: program

File: docs/ballerina-by-example/examples/grpc-server-streaming/client/helloWorld.sample.client.bal [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 int total = 0;
 function main (string[] args) {
-    // Client endpoint configuration
+    // The client endpoint configuration.
     endpoint HelloWorldClient helloWorldEp {
         host:"localhost",
         port:9090
     };
-    // Executing unary non-blocking call registering server message listener.
+    // The executing unary non-blocking call registering server message listener.
```
Comment (afkham) [inline docs/ballerina-by-example/examples/grpc-server-streaming/client/helloWorld.sample.client.bal]: This doesn't sound correct

File: docs/ballerina-by-example/examples/grpc-unary-blocking/client/helloWorld.sample.client.bal [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
 function main (string[] args) {
-    // Client endpoint configuration
+    // The client endpoint configuration.
     endpoint HelloWorldBlockingClient helloWorldBlockingEp {
         host:"localhost",
         port:9090
     };
 
-    // Executing unary blocking call
+    // The executing unary blocking call.
```
Comment (afkham) [inline docs/ballerina-by-example/examples/grpc-unary-blocking/client/helloWorld.sample.client.bal]: This doesn't sound correct

File: docs/ballerina-by-example/examples/grpc-unary-non-blocking/client/helloWorld.sample.client.bal [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
 int total = 0;
 function main (string[] args) {
-    // Client endpoint configuration
+    // The client endpoint configuration.
     endpoint HelloWorldClient helloWorldEp {
         host:"localhost",
         port:9090
     };
-    // Executing unary non-blocking call registering server message listener.
+    // The executing unary non-blocking call registering server message listener.
```
Comment (afkham) [inline docs/ballerina-by-example/examples/grpc-unary-non-blocking/client/helloWorld.sample.client.bal]: This doesn't sound correct

File: docs/ballerina-by-example/examples/hello-world-parallel/hello-world-parallel.description [hunk 9]
```diff
@@ -1,2 +1,2 @@
-//Let's print "Hello, World!" in parallel using Ballerina.
-//Here we use workers which represent parallel threads of execution in Ballerina.
\ No newline at end of file
+//Let's print 'Hello, World!' in parallel using Ballerina.
+//Here, the workers used represent parallel threads of execution in Ballerina.
```
Comment (afkham) [inline docs/ballerina-by-example/examples/hello-world-parallel/hello-world-parallel.description]: Here, the workers represent parallel executions in Ballerina.

File: docs/ballerina-by-example/examples/http-redirects/http-redirects.description [hunk 10]
```diff
@@ -1,2 +1,2 @@
-//To follow redirects when calling an external http server using ballerina http client connector set "followRedirect"
-//to true.
+//To follow redirects when calling an external http server using the ballerina http client connector, set the 'followRedirect' property
```
Comment (afkham) [inline docs/ballerina-by-example/examples/http-redirects/http-redirects.description]: s/ballerina/Ballerina
s/http/HTTP

File: docs/ballerina-by-example/examples/http-load-balancer/http-load-balancer.server.sh [hunk 11]
```diff
@@ -1,6 +1,6 @@
-# To run the service, put the code in `http-load-balancer.bal`
-# and execute the following command from `$BALLERINA_HOME/bin`.
+# To run the service, put the code in the 'http-load-balancer.bal' file
+# and execute the following command from the '$BALLERINA_HOME/bin' director.
```
Comment (uthaiyashankar) [inline docs/ballerina-by-example/examples/http-load-balancer/http-load-balancer.server.sh]: directory..

**Comment by uthaiyashankar** [comment]
Ballerina by example is moved to https://github.com/ballerina-platform/ballerina-examples. Please send the PR to the new repo.

**Pull request closed without merging**